README.md

```diff
@@ -48,12 +48,29 @@ Some NFC drivers depend on third party software:
 The regression test suite depends on the cutter framework:
 http://cutter.sf.net
 
+Building
+========
+
+Note: If working directly from a git clone of the repository, some of the files need to be generated first. To do this run
+`autoreconf -vls`
+
+Alternatively use a tar.gz or zip version of a packaged release release (which already contains ./configure):
```

Comment:
Only the libnfc-x.y.z.tar.bz2 file contains the generated autoconf file.
The .zip archive and .tar.gz archive does NOT contain it.

Reply:
Fixed